### PR TITLE
Remove DeprecatedDownwardAPIVolumeSource from protobuf

### PR DESCRIPTION
**Pull request by ncdc**
@smarterclayton PTAL

### CODE REVIEW

**Comment by ncdc** [comment]
If this looks good, I have similar patches for 1.3 and 1.4.

**Comment by smarterclayton** [comment]
Lgtm.

On Nov 22, 2016, at 12:56 PM, Andy Goldstein <notifications@github.com>
wrote:

If this looks good, I have similar patches for 1.3 and 1.4.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/openshift/origin/pull/11997#issuecomment-262315131>,
or mute
the thread

**Comment by ncdc** [comment]
Fixed commit message

**Comment by ncdc** [comment]
[test]

**Comment by ncdc** [comment]
@liggitt @deads2k @sttts FYI

**Comment by ncdc** [comment]
Something's up with the round tripping. Will investigate today.

File: vendor/k8s.io/kubernetes/pkg/api/serialization_proto_test.go [hunk 1]
```diff
@@ -70,6 +70,11 @@ func TestProtobufRoundTrip(t *testing.T) {
 	// InitContainers are turned into annotations by conversion.
 	obj.Spec.InitContainers = nil
 	obj.Status.InitContainerStatuses = nil
+	// We don't round trip VolumeSource.Metadata (it's a deprecated OpenShift
```
Comment (deads2k) [inline vendor/k8s.io/kubernetes/pkg/api/serialization_proto_test.go]: lacks beauty, but it'll get the job done.

**Comment by ncdc** [comment]
@smarterclayton PTAL at the change to serialization_proto_test.go

**Comment by ncdc** [comment]
ok to merge?

**Comment by deads2k** [comment]
`Remove DeprecatedDownwardAPIVolumeSource from protobuf` should be a carry, then lgtm.

**Comment by ncdc** [comment]
Commit message updated

**Comment by ncdc** [comment]
[merge]

**Comment by ncdc** [comment]
#11004 #11775 [merge]

**Comment by ncdc** [comment]
Some weird vagrant flake

```
+ vagrant sync-origin -s --dont-install
/var/lib/jenkins/.vagrant.d/gems/gems/vagrant-openshift-3.0.8/lib/vagrant-openshift/action/prepare_ssh_config.rb:39:in `initialize': Text file busy - /var/lib/jenkins/.ssh/vagrant_openshift_ssh_override (Errno::ETXTBSY)
	from /var/lib/jenkins/.vagrant.d/gems/gems/vagrant-openshift-3.0.8/lib/vagrant-openshift/action/prepare_ssh_config.rb:39:in `open'
	from /var/lib/jenkins/.vagrant.d/gems/gems/vagrant-openshift-3.0.8/lib/vagrant-openshift/action/prepare_ssh_config.rb:39:in `call'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/warden.rb:34:in `call'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/builder.rb:116:in `call'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/runner.rb:66:in `block in run'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/util/busy.rb:19:in `busy'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/action/runner.rb:66:in `run'
	from /var/lib/jenkins/.vagrant.d/gems/gems/vagrant-openshift-3.0.8/lib/vagrant-openshift/command/repo_sync_origin.rb:63:in `block in execute'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/plugin/v2/command.rb:226:in `block in with_target_vms'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/plugin/v2/command.rb:220:in `each'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/plugin/v2/command.rb:220:in `with_target_vms'
	from /var/lib/jenkins/.vagrant.d/gems/gems/vagrant-openshift-3.0.8/lib/vagrant-openshift/command/repo_sync_origin.rb:61:in `execute'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/cli.rb:42:in `execute'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.7.4/lib/vagrant/environment.rb:301:in `cli'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.7.4/bin/vagrant:174:in `<main>'
Build step 'Execute shell' marked build as failure
```

[merge]

**Pull request merged**